src/app/app.component.html

```diff
@@ -1,6 +1,8 @@
-<!--The content below is only a placeholder and can be replaced.-->
 
-<!--Router-outlet-->
-<router-outlet>
-</router-outlet>
-<!--Router-outlet-->
+
+<ul>
+<li><a href="#Home">Home</a></li>
+<li><a href="#Dashboard">Dashboard</a></li>
```

Comment:
does this works?

Reply:
This redirects me to http://localhost:4200/#Home or http://localhost:4200/#Dashboard, but they don't contain anything. Should I delete them?